torch/onnx/__init__.py

```diff
@@ -305,7 +305,8 @@ def forward(self, x):
 
         enable_onnx_checker (bool, default True): If True the onnx model checker will be run
             to ensure the exported model is a valid ONNX model.
-        use_external_data_format (bool, default False): If True, then some of the model
+        use_external_data_format (bool, default False): [DEPRECATED, check if models are larger than 2GB
```

Comment:
[DEPRECATED, check if models are larger than 2GB in backend to set this parameter to True or False]

changed to :

[Deprecated and ignored. Will be removed in next Pytorch release.]